File: MODUL 7/SOAL 1/PRAK601-2210817310002-MUHAMMADRIDHOPRASETYA.c
#include <stdio.h>

int main(){
    int baris,kolom,i,j,isi;
    scanf("%d %d",&baris,&kolom);
    int angka[baris][kolom];
        for(i=0; i<baris; i++){
    for(j=0; j<kolom; j++){
    scanf("%d",&isi);
    angka[i][j]=isi;
    }
}
for(i=0; i<baris; i++){
    for(j=0; j<kolom; j++){
        printf("%d ", angka[i][j]);
    }
    printf("\n");
}
}

File: MODUL 7/SOAL 1/PRAK601-2210817310002-MUHAMMADRIDHOPRASETYA.py
a, b = map(int, input().split())
angka = []
x = 0
isi = list(map(int, input().split()))
for i in range (a):
    angka.append(isi[x : x + b])
    x = x + b
for a in angka:
    for isi in a:
        print(isi, end=' ')
    print()